ruby/vampires.rb

```diff
@@ -0,0 +1,97 @@
+# Methods to check user input
+def check_if_integer
+  begin
+    num = Integer(gets.chomp)  
+  rescue 
+    puts 'Error reading input - please enter an integer number:'
+    retry
+  end
+
+  num
+end
+
+def yes_or_no
+  loop do
+    case gets.chomp.downcase
+    when 'yes'
+      return 'yes'
```

Comment:
Ruby is an implicit language so you do not need the returns here.  Ruby will implicitly return the last thing it reads in a statement.

Nothing about this method is incorrect but stylistically look into explicit and implicit returns in Ruby and let me know if you have questions.